src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends ColorSupplier {
+    public Ball getRandomBall() {
+
+        Ball ball = new Ball();
+
+        ball.setBallColor(ColorSupplier.getRandomColor());
+
+        ball.setBallNumber(new Random().nextInt(100));
+
+        return ball;
+    }
+}
```

Comment:
As in general opinion, wanna add that, maybe we need to move getting random number to separate class. We do it twice, but can do it with methods and variables. What if in future we need range "from a to b" to provide random numbers. Maybe, some class like `NumbersSupplier`.